player/lua/osc.lua

```diff
@@ -73,15 +73,15 @@ local user_opts = {
     -- luacheck: max line length
     playlist_prev_mbtn_left_command = "playlist-prev",
     playlist_prev_mbtn_mid_command = "show-text ${playlist} 3000",
-    playlist_prev_mbtn_right_command = "show-text ${playlist} 3000",
+    playlist_prev_mbtn_right_command = "script-binding select/select-playlist; script-message-to osc osc-hide",
 
     playlist_next_mbtn_left_command = "playlist-next",
     playlist_next_mbtn_mid_command = "show-text ${playlist} 3000",
-    playlist_next_mbtn_right_command = "show-text ${playlist} 3000",
+    playlist_next_mbtn_right_command = "script-binding select/select-playlist; script-message-to osc osc-hide",
 
     title_mbtn_left_command = "script-binding stats/display-page-5",
     title_mbtn_mid_command = "show-text ${filename}",
-    title_mbtn_right_command = "script-binding select/select-playlist; script-message-to osc osc-hide",
+    title_mbtn_right_command = "show-text ${path}",
```

Comment:
Should this be $path or $stream-path or $stream-open-file?

Reply:
Never heard of stream-path, docs say it's useless. stream-open-filename is the huge EDL string when playing ytdl_hook URLs I don't know why you would want to see that instead of the normal youtube URL.